main.go

```diff
@@ -37,6 +37,8 @@ import (
 
 	lvmv1alpha1 "github.com/openshift/lvm-operator/api/v1alpha1"
 	"github.com/openshift/lvm-operator/controllers"
+	persistent_volume "github.com/openshift/lvm-operator/controllers/persistent-volume"
+	persistent_volume_claim "github.com/openshift/lvm-operator/controllers/persistent-volume-claim"
```

Comment:
Same here. Any reason not to use `pvc` instead of `persistent_volume_claim`?

Reply:
For the same reason above, I will keep it as `persistent_volume_claim`.